integration_tests/macros/e2e_tests/tests_validation.sql

```diff
@@ -4,7 +4,7 @@
     {% set dimension_validation_query %}
         select *
         from {{ alerts_relation }}
-        where status in ('fail', 'warn') and sub_type = 'dimension'
+        where status in ('fail', 'warn') and tags like '%dimension_anomalies%'
```

Comment:
The tests for `directional_anomalies` (and `schema`) weren't running before.
When they did run the results were mixed with the dimension results, so I made changes to make sure each will be tested separately.